mysite/settings.py

```diff
@@ -78,20 +78,28 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.\
-            UserAttributeSimilarityValidator',
+        'NAME': (
```

Comment:
This PR includes `bee985698e79d2857fb50240dce0beb05f6315c0`, probably because it's not rebased on top of `origin/master`